graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlErrorsHandler.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.graphql;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.common.annotation.UnstableApi;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import graphql.ExecutionInput;
+import graphql.ExecutionResult;
+
+/**
+ * A handler that maps {@link ExecutionResult#getErrors()} or the specified {@link Throwable}
+ * to an {@link HttpResponse}.
+ */
+@UnstableApi
+@FunctionalInterface
+public interface GraphqlErrorsHandler {
```

Comment:
Even though `ExecutionResult` returns a list of errors, it is a bit weird to use Error`s` in the handler. How about simply naming it `GraphqlErrorHandler`?
```suggestion
@UnstableApi
@FunctionalInterface
public interface GraphqlErrorHandler {
```

Reply:
Is it better to change the name of the interface but keep the explanation of javadoc?

Reply:
I am OK with keeping the Javadoc.

Reply:
> Is it right to change all these names?

Right apart from `GraphqlErrorsHandlers`. I prefer to rename `GraphqlErrorsHandlers` to  `DefaultGraphqlErrorsHandler`. 
For example:
```
enum DefaultGraphqlErrorHandler implements GraphqlErrorHandler {
    INSTANCE;

    // Also override nullability with `@Nonnull` so that the default error handler never returns null.
    @Nonnull
    @Override
    HttpResponse handle(...) {
        ...
    }
    
}
```

Reply:
I've completed the implementation and committed it!

I knew that enum could be used for this purpose, but this is the first time I have actually implemented it. So I'm worried if it's properly implemented. 🤔 Especially, I searched hard, but I don't know if Enum's instance is static final.

https://github.com/line/armeria/blob/58f402c40389a0f18f9e26725fd79d8c4179cb7e/graphql/src/main/java/com/linecorp/armeria/server/graphql/DefaultGraphqlErrorHandler.java#L32-L34

Anyway, I'm so happy to learn a lot from the advice I received while contributing to this issue that I couldn't learn while studying alone with books. 😀